kornia/augmentation/container/ops.py

```diff
@@ -0,0 +1,390 @@
+from abc import ABCMeta, abstractmethod
+from typing import Any, Callable, Dict, Generic, List, Optional, Type, TypeVar, Union
+
+import kornia
+from kornia.augmentation import GeometricAugmentationBase2D, GeometricAugmentationBase3D, MixAugmentationBaseV2
+from kornia.augmentation.base import _AugmentationBase
+from kornia.augmentation.container.base import ParamItem
+from kornia.constants import DataKey
+from kornia.core import Module, Tensor
+from kornia.geometry.boxes import Boxes
+from kornia.geometry.keypoints import Keypoints
+
+DataType = Union[Tensor, List[Tensor], Boxes, Keypoints]
+
+T = TypeVar('T')
+
+
+class SequentialOpsInterface(Generic[T], metaclass=ABCMeta):
```

Comment:
To mention that, I think this file follows the best practices for the typing stuff. Probably we shall somehow refactor the whole augmentation codebase similarly. One nasty typing issue is that we now have 'Boxes', 'Keypoints', etc.

Reply:
Yes, if we go to drop the type: ignore[override], i think we will need to do a refactor in this way -- my initial intention is to keep the override cases and rip out the pure type ignore cases (which are the ones that can cause the most problems)